docs/TODO.md

```diff
@@ -12,8 +12,8 @@ point on the checklist for the exam.
 - [x] Fill out the `make_dataset.py` file such that it downloads whatever data you need and 
 - [x] Add a model file and a training script and get that running
 - [x] Remember to fill out the `requirements.txt` file with whatever dependencies that you are using
-- [ ] Remember to comply with good coding practices (`pep8`) while doing the project
-- [ ] Do a bit of code typing and remember to document essential parts of your code
+- [x] Remember to comply with good coding practices (`pep8`) while doing the project
+- [x] Do a bit of code typing and remember to document essential parts of your code
```

Comment:
I think the documentation is still a todo, while I think you did this: _(extra) Implemented pre-commit hooks for your project repository_